build.sbt

```diff
@@ -168,6 +170,7 @@ lazy val `alpakka-kafka` =
     .settings(commonSettings)
     .settings(
       skip in publish := true,
+      // TODO: how do we add clusterSharding to unidoc and only build it with Akka 2.6?
```

Comment:
We would need to build the docs against Akka 2.6.
Leave it for now.

Reply:
Ok.  I left a note to add it when we drop 2.5.